server/controllers/conversation.js

```diff
@@ -0,0 +1,67 @@
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /conversations
+// @desc Create new conversation
+// @access Public
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { dogOwnerId, dogSitterId } = req.body;
+
+  try {
+    const conversation = await Conversation.create({
```

Comment:
Let's check if a conversation between two users exists before we create a new one :)